src/modules/sway/workspaces.cpp

```diff
@@ -102,13 +121,27 @@ void Workspaces::onCmd(const struct Ipc::ipc_response &res) {
                     // the "num" property (integer type):
                     // The workspace number or -1 for workspaces that do
                     // not start with a number.
-                    auto l = lhs["num"].asInt();
-                    auto r = rhs["num"].asInt();
+                    //auto l = lhs["num"].asInt();
+                    //auto r = rhs["num"].asInt();
```

Comment:
Shouldn't we keep it as a fallback?

Reply:
Maybe, but this way waybar is completely independent from how sway assigns the `num` property resulting in less unexpected behaviour in waybar, in case sway ever changes the way the `num` property is set.